sycl/include/sycl/device_selector.hpp

```diff
@@ -16,19 +16,19 @@
 __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 
+// SYCL 1.2.1 defines a negative score to reject a device from selection
+static constexpr int REJECT_DEVICE_SCORE = -1;
```

Comment:
Should we put this into detail namespace?

Reply:
yes. done.